.github/workflows/containers.yml

```diff
@@ -208,8 +202,7 @@ jobs:
           source test/container/test-fsim-wget.sh
           run_test
 
-      - name: Get manufacturer, rendezvous and owner server logs after failure
-        if: failure()
+      - name: Get manufacturer, rendezvous and owner server logs
```

Comment:
I also don't understand why we don't capture all logs by default too - pretty easy, not expensive, and allows us to check stuff out on positive runs too...

Reply:
https://github.com/fido-device-onboard/go-fdo-server/issues/109